pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
 
-**Getting Started**
-For a detailed **walkhrough of getting started** with build, we will point you to the <a href="/quickstart">quickstart</a>. 
+{% include toc.html %}
 
-**Build Environment**
-If you want to customize the cache, specify Docker credentials, or any custom tweaks to your build environment, see <a href="/build-environment">build environment</a>. 
+## Overview
+The `build` command accepts a target as input and produces a container as output.  
 
-**Modular Containers**
-If you want to look at how to get started to making interally **modular containers**, check out <a href="http://containers-ftw.org/apps/examples/tutorials/getting-started" target="_blank">this post</a>.
+The target defines the method that `build` uses to create the container. It can be one of the following:
 
+- URI beginning with **shub://** to build from Singularity Hub
+- URI beginning with **docker://** to build from Docker Hub
+- path to a **existing container** on your local machine
+- path to a **directory** to build from a sandbox
+- path to an **archive** in .tar or compressed .tar.gz format
+- path to a **[Recipe file](docs-recipes)**
 
-{% include toc.html %}
+In addition `build` can produce containers in three different formats.  Formats types can be specified by passing the following options to build.  
 
-## Bases
-You have many options for the base of your build. See the linked page for each for more details, or read the [Usage](#usage) section below for a summary.
+- compressed read-only **squashfs** file system suitable for production (default)
+- writable **ext3** file system suitable for interactive development (`--writable` option)
+- writable **(ch)root directory** called a sandbox for interactive development (`--sandbox` option)
 
-### Singularity Recipes
-For a reproducible container, the recommended practice is to build by way of a Singularity recipe file. This also makes it easy to add files, environment variables, and install custom software, and still start from your base of choice (e.g., Docker). The absolute minimum required for a recipe is a base, and here are your options:
+Because `build` can accept an existing container as a target and create a container in any of these three formats you can convert existing containers from one format to another.
 
+Some things need root...
 
-**Singularity Hub**
-```
-Bootstrap: shub
-From: vsoch/hello-world
-```
+## Downloading a pre-built container from Singularity Hub
+You can use the `build` command to download a container from Singularity Hub. 
 
-[**Docker**](/docs-docker)
```

Comment:
This still doesn't make sense to me. If you are using build with singularity hub, then you are creating a new image FROM one on singularity hub. If that is incorrect, then it should be  a pull. But saying "download container" and then saying we are building is really confusing for the user.